docs/components/encoder/single-micro-rdk.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Configure a Single Encoder (Micro-RDK)"
+title: "Single Encoder (viam-micro-server)"
```

Comment:
Why is the "Configure a" going? If the title is too long lmk and we may need to oberweite that setting

Reply:
Yeah it’s because the title is too long for lots of the micro server ones. So I made the same change to all micro components of a given subtype so it’d at least be semi-uniform. Would you be okay with changing the setting to be more lax everywhere? Or is it there for a reason?